js/FileManager.js

```diff
@@ -0,0 +1,156 @@
+var InsightMakerFileExtension = ".InsightMaker";
+
+// Append file extension to file (if not already there)
+function append_file_extension(filename,extension) {
+	var extension_position=filename.length-extension.length;
+	var current_extension=filename.substring(extension_position,filename.length);
+	if(current_extension.toLowerCase()!=extension.toLowerCase()) {
+		filename+=extension;
+	}
+	return filename;
+}
+
+// Set the title to include the model name
+function set_title(filename) {
+	var title;
+	if(filename!=null) {
+		title = filename+"|InsightMaker";
+		
+	} else {
+		title = "InsightMaker";
+	}
+	window.parent.document.title = title;
+}
+
+// Get xml data for the current model
+function get_model_xml() {
```

Comment:
Yes. My fault. I started programming mostly with c, php and python and there camelCase is rarely used. I should fix my habbits for this project!